relayer/middleware/missedVaas.middleware.ts

```diff
@@ -104,24 +104,19 @@ export function missedVaas(
         "No missed VAAs detected between this VAA and the last VAA we processed."
       );
     }
-    await redisPool.release(redis);
     try {
-      await next(); // <-- process the current vaa
+      await setLastSequenceForContract(
```

Comment:
wouldn't this cause some VAAs to be possibly missed?
In the case that the processing of the VAA fails, then this VAA won't be fetched by missed VAA fetcher, right? or am I missing something?